docs/pages/management/operations/upgrading.mdx

```diff
@@ -49,6 +49,11 @@ Back up before upgrading. We have more instructions in [Backing up Teleport](./b
 
 ## Upgrade Sequence
 
+When upgrading multiple clusters:
```

Comment:
I'm not sure moving this first makes sense, as trusted clusters are a more complicated setup. My expectation would be that those who come to this doc are not familiar with the upgrade process, and therefore probably not yet at the point where they're running trusted clusters.